clang/lib/Sema/SemaAttr.cpp

```diff
@@ -269,6 +271,40 @@ void Sema::inferLifetimeBoundAttribute(FunctionDecl *FD) {
   }
 }
 
+static bool IsPointerLikeType(QualType QT) {
+  QT = QT.getNonReferenceType();
+  // if (QT->isPointerType())
```

Comment:
any reason to comment it out? I think having this should fix some FIXMEs in the test.